CHANGELOG.md

```diff
@@ -34,6 +34,9 @@ should change the heading of the (upcoming) version to include a major version b
 - For JSON Schemas with `$id`s, use a pre-compiled Ajv validation function when available.
 - No longer fail to validate inner schemas with `$id`s, fixing [#2821](https://github.com/rjsf-team/react-jsonschema-form/issues/2181).
 
+## @rjsf/antd
+- No longer render extra 0 for array without errors, fixing [#3233](https://github.com/rjsf-team/react-jsonschema-form/issues/3233)
+
```

Comment:
Can you move this up higher in the file under the `beta.14` section

Reply:
I'm sorry I just saw it now. I'd be happy to go and fix it, but it's a pity you've already merged. I think I'll check the notifications more often in the future.

Reply:
No worries, I was putting in a fix of my own and fixed up the `CHANGELOG.md` as part of that change

Reply:
I see it. Thanks.